packages/server/graphql/public/types/NotifyDiscussionMentioned.ts

```diff
@@ -0,0 +1,21 @@
+import MeetingTeamPrompt from '../../../database/types/MeetingTeamPrompt'
+import {NotifyDiscussionMentionedResolvers} from '../resolverTypes'
+
+const NotifyDiscussionMentioned: NotifyDiscussionMentionedResolvers = {
+  __isTypeOf: ({type}) => type === 'DISCUSSION_MENTIONED',
+  meeting: async ({meetingId}, _args, {dataLoader}) => {
+    const meeting = await dataLoader.get('newMeetings').load(meetingId)
+    return meeting as MeetingTeamPrompt
+  },
+  author: ({authorId}, _args: unknown, {dataLoader}) => {
+    return dataLoader.get('users').loadNonNull(authorId)
+  },
+  comment: ({commentId}, _args: unknown, {dataLoader}) => {
+    return dataLoader.get('comments').load(commentId)
```

Comment:
+1 comment isn't nullable, should we use `loadNonNull` here as well?

Reply:
The `comments` dataloader is already non-nullable - `load` returns a non-null Promise, and it doesn't have a `loadNonNull` method.

Reply:
ah yeah, this'll be fixed when we move away from rethinkdb. the rethinkdb PK loaders don't use the new pattern so it assumes everything is non-null, even when that's not true